usr/share/rear/restore/BAREOS/default/410_restore_backup_bconsole.sh

```diff
@@ -0,0 +1,80 @@
+#
+# Restore from Bareos using bconsole (optional)
+#
+
+if [ "$BAREOS_RESTORE_MODE" != "bconsole" ]; then
+    return
+fi
+
+# get last restore jobid before starting the restore
```

Comment:
This is used to determine if a new restore job has been started. Before we start a restore, we query the number of the last restore job (if any). Then we start the restore process, and check if the last known restore job differs from the one we have stored. If it differs, if assume that this is the restore job started by us.
If our restore script do start the restore job, there are also other ways that to get the jobid of the started restore job. However, this way it also works if the user manually starts the restore job, or even if the restore job is triggered from the server.

Reply:
When the user starts the restore manually then ReaR won't have to watch for it. If starting the job on the server is a thing then besides `automatic` and `manual` there should be a third option `server` to denote "wait for the restore to magically start and finish".

So in the `automatic` case we should be able to handle the job ID explicitly without guessing. In `manual` ReaR does *not* monitor the progress at all and in `server` ReaR should wait either for a clear signal or manual input to continue, as it can't know if there is only one or multiple restore jobs required.

I'm saying this because the job of ReaR is to be clear when the user can rely on it and to be clear when not. Guessing stuff always adds a big risk of additional ambiguity which is harmful in the stressful situation of a disaster recovery. So better do less but be 100% sure about it and hand the decisions to the user if we are not sure.